float32 pitch
float32 yaw
float32 pitch_speed
float32 yaw_speed
